web_src/js/features/repo-home.js

```diff
@@ -16,6 +16,12 @@ export function initRepoTopicBar() {
     viewDiv.hide();
     editDiv.css('display', ''); // show Semantic UI Grid
   });
+  mgrBtn.on('keydown', (e) => {
+    if (e.key === 'Enter' || e.key === ' ') {
+      viewDiv.hide();
+      editDiv.css('display', ''); // show Semantic UI Grid
```

Comment:
We could think about extracting the function body (without condition) into a separate method…

Reply:
Maybe use a `button`, then it can automatically handle `enter` as click.